data.sql

```diff
@@ -6,4 +6,15 @@ VALUES
     ('Gabumon', '2018-11-15', 2, true, 8),
     ('Pikachu', '2021-01-07', 1, false, 15.04),
     ('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
```

Comment:
- [ ] Great job working on this milestone. The data you are to insert has the following fields `name, date_of_birth, escape_attempts, neutered, weight_kg`. Your inserted data has the `species` field which is not a requirement. Kindly update your fields.

Reply:
Ohh boy!! .... You're reason, sir .... I fixed the script in my **data.sql** file

![FIX-01](https://github.com/jlcarrascof/Vet-clinic-database/assets/86327867/c18c50b9-f691-4545-9107-94fa483133c1)